src/Home/Home.spec.tsx

```diff
@@ -95,27 +97,73 @@ describe("Home", () => {
       const proximityTracingStatusContainer = getByTestId(
         "home-proximity-tracing-status-container",
       )
+      const bluetoothEnabledText = within(bluetoothStatusContainer).getByText(
+        "Enabled",
+      )
 
-      await waitFor(() => {
-        const bluetoothEnabledText = within(bluetoothStatusContainer).getByText(
-          "Enabled",
-        )
-        const proximityTracingEnabledText = within(
-          proximityTracingStatusContainer,
-        ).getByText("Enabled")
+      const proximityTracingEnabledText = within(
+        proximityTracingStatusContainer,
+      ).getByText("Enabled")
 
-        expect(header).toHaveTextContent("Active")
-        expect(subheader).toHaveTextContent(
-          `${mockedApplicationName} will remain active after the app has been closed`,
-        )
-        expect(bluetoothEnabledText).toBeDefined()
-        expect(proximityTracingEnabledText).toBeDefined()
-      })
+      expect(header).toHaveTextContent("Active")
+      expect(subheader).toHaveTextContent(
+        `${mockedApplicationName} will remain active after the app has been closed`,
+      )
+      expect(bluetoothEnabledText).toBeDefined()
+      expect(proximityTracingEnabledText).toBeDefined()
+    })
+  })
+
+  describe("When the exposure notification permissions are enabled and the app is authorized, Bluetooth is on, and Location is off, but the platform is iOS", () => {
```

Comment:
this is getting a bit long of a title, what do you think about something more like
```suggestion
  describe("When the device platform is iOS, () => {
    describe("and the exposure notification permissions are enabled and the app is authorized, Bluetooth is on, and Location is off", () => {
```

And wouldnt the location not affect the behavior for ios?

Reply:
Yes, I was thinking that this would test that. Does that make sense?

Reply:
Ended up removing this test because it was testing the same thing as the final tests.